src/App.jsx

```diff
@@ -5,19 +5,23 @@ import { Route, Routes } from 'react-router-dom'
 import './App.css'
 import Header from './Components/Header/Header'
 import ArticleList from './Components/Articles/ArticleList'
+import ArticleById from './Components/Articles/ArticleById'
+import Comments from './Components/Comments/Comments'
 
 function App() {
   const [loggedInUser, setLoggedInUser] = useState({
```

Comment:
I would strongly recommend setting the logged in user as an actual user to avoid problems with posting (unless you specifically want there to be an issue with them posting)